examples/default-exchange/config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    vhosts: {
+        "/": {
+            connection: {
+                heartbeat: 1,
+                socketOptions: {
+                    timeout: 1000
+                }
+            },
+            exchanges: [""],
```

Comment:
It would be nice to always add the default exchange to the list of exchanges (unless explicitly specified)